lib/http/request.rb

```diff
@@ -66,20 +66,24 @@ class UnsupportedSchemeError < RequestError; end
     # Scheme is normalized to be a lowercase symbol e.g. :http, :https
     attr_reader :scheme
 
+    attr_reader :uri_normalizer
```

Comment:
No need to expose it.

Reply:
I am exposing it because we are using it in `wrap_request` method in `auto_deflate`